nipoppy_cli/tests/test_tabular_bagel.py

```diff
@@ -11,15 +11,15 @@
     [
         {
             Bagel.col_participant_id: "1",
-            Bagel.col_bids_id: "sub-1",
-            Bagel.col_session: "ses-01",
+            Bagel.col_bids_participant: "sub-1",
+            Bagel.col_session_id: "01",
```

Comment:
Doesn't really matter, but until now you were using `session_id` without zero padding.

Reply:
Yeah I don't think we have to enforce a specific style for the tests. More variety is usually good I think